crates/storage/src/merkle/column.rs

```diff
@@ -56,7 +63,11 @@ where
             Self::MerkleDataColumn(column) => {
                 Self::MERKLE_DATA_COLUMNS_START.wrapping_add(column.as_u32())
             }
-            Self::MerkleMetadataColumn => u32::MAX,
+            Self::MerkleMetadataColumn => Self::COUNT
+                .checked_sub(1)
+                .expect("qed")
+                .try_into()
+                .expect("qed"),
```

Comment:
Hmm.. I don't like having panics here but I don't see an obvious way around it. First of all, I think "qed" is an unhelpful message so instead we can just unwrap the values.

Reply:
```suggestion
            Self::MerkleMetadataColumn => Self::COUNT
                .checked_sub(1)
                .unwrap()
                .try_into()
                .unwrap()
```